core/make_binders.py

```diff
@@ -33,7 +33,6 @@ class MethodBind$argc$$ifret R$$ifconst C$ : public MethodBind {
 	}
 
 	virtual Variant call(Object* p_object,const Variant** p_args,int p_arg_count, Callable::CallError& r_error) {
-
```

Comment:
The `.py` files likely need a manual clang-formatting overall, there are lots of style issues.

Reply:
(By manual I mean it should be done by a contributor visually, we can't run clang-format on Python files (one could trick it by copying the content in a `.cpp` file, format it, and then copy back).